src/components/info/byline.js

```diff
@@ -101,6 +102,22 @@ function renderMaintainers(t, metadata) {
   return null;
 }
 
+/**
+ * Returns a React component detailing the date the data was last updated.
+ * Renders a <span> containing "Data updated X", where X derives from `metadata.updated`
+ */
+function renderDataUpdated(t, metadata) {
+  if (metadata.updated) {
+    return (
+      <span>
+        {t("Data updated") + " "}
```

Comment:
This can be more succinctly written via
```jsx
<span>
  {`${t('Data updated')} ${metadata.updated}.`}
</span>  
```
(Up to you - it's stylistic)

Reply:
Good suggestion! Done in [force-push](https://github.com/nextstrain/auspice/compare/67d8806100945d3c7dc339addd6249561e7d9bc5..60815b4eaca60d3721f52ae3228ee8b303709488).

I prefer interpolated strings in general. Added a space after the `.` to be consistent with other entries.